db/migrate/20160419134542_add_deployment_authentication_options_to_authentications.rb

```diff
@@ -0,0 +1,35 @@
+class AddDeploymentAuthenticationOptionsToAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    add_column :authentications, :auth_challenge, :boolean
+    add_column :authentications, :auth_login, :boolean
+    # from old table t.string :kind assuming we could use auth_type is the correct? (could be ldap/htpass etc..)
+    add_column :authentications, :auth_public_key, :text #for supporting ssh public key (needed on deployment model)
+    add_column :authentications, :htpassd_users, :text, :array => true, :default => []
+    add_column :authentications, :ldap_id, :text, :array => true, :default => []
+    add_column :authentications, :ldap_email, :text, :array => true, :default => []
+    add_column :authentications, :ldap_preferred_user_name, :text, :array => true, :default => []
+    add_column :authentications, :ldap_bind_dn, :string
+    add_column :authentications, :ldap_bind_password, :string #should i store this as password?
```

Comment:
I sort of like having it scoped to `ldap`, but I can see it going either way.

@Fryguy your preference here?